crates/bootstrap_srv/src/server.rs

```diff
@@ -68,7 +71,10 @@ impl BootstrapSrv {
 
         // get the address that was assigned
         let addrs = server.server_addrs().to_vec();
-        println!("Listening at {:?}", addrs);
+        for addr in addrs.iter() {
+            // print these separately incase someone wants to parse them
+            tracing::info!(addr = format!("{:?}", addr), "Listening");
```

Comment:
Would that be easier if they went to stdout rather than out with the tracing?

Reply:
:+1: Yeah, that's probably better. Will update.

Reply:
https://github.com/holochain/kitsune2/pull/50/commits/b063bf1c9d044f41e9cedc681e1081f8048b16b7